### ISSUE TITLE
Move simple content into the ContentInput component

### ISSUE THREAD

**Issue by bngarren** [bngarren/icu-rounder#21]
- The **ToggleContentType** component should remain separate from ContentInput
- The **CustomFormControlEditor** components for "simple" and "nested" content should be moved into **ContentInput**
- The contentType should be passed as a prop to **ContentInput**, which then renders conditionally based on this

**Old code:** 

```javascript
<div>
    <CustomFormControlEditor
      id="contentType"
      initialValue={defaultValues.contentType || "simple"}
      onInputChange={handleOnToggleContentType}
      onDiffChange={onDiffChange}
      onChangeArgument={1}
    >
      <ToggleContentType />
    </CustomFormControlEditor>
    {(data.contentType == null || data.contentType === "simple") && (
      <CustomFormControlEditor
        id="simpleContent"
        initialValue={defaultValues.simpleContent || ""}
        onInputChange={handleInputChange}
        onDiffChange={onDiffChange}
        onBlur={handleInputOnBlur}
      >
        <CustomTextField
          className={classes.textFieldBody}
          label="Content"
          variant="filled"
          multiline
          rows={10}
          customStyle={classes}
        />
      </CustomFormControlEditor>
    )}
    {data.contentType === "nested" && (
      <CustomFormControlEditor
        id="nestedContent"
        initialValue={defaultValues.nestedContent || ""}
        onInputChange={handleInputChange}
        onDiffChange={onDiffChange}
        onBlur={handleInputOnBlur}
        onChangeArgument={1}
      >
        {/* also pass the initialValue prop here so that ContentInput
        knows when to reset itself, i.e. after a bed change */}
        <ContentInput
          initialValue={defaultValues.nestedContent || ""}
        />
      </CustomFormControlEditor>
    )}
</div>
```

**Proposal:**

*BedspaceEditor.js*
```javascript
    <CustomFormControlEditor>
      <ToggleContentType />
    </CustomFormControlEditor>
    <ContentForm contentType={contentType} onChange={handleInputChange}/>
```
*ContentInput.js*
```javascript
    <CustomFormControlEditor>
      <SimpleContent />
    </CustomFormControlEditor>
    <CustomFormControlEditor>
      <NestedContent />
    </CustomFormControlEditor>
```

**Comment by bngarren** [comment]
The bulk of the issue is addressed with 10448e6. 

- [ ] Need to review code quality once more and add documentation
- [ ] Finish styling of ContentInput

**Comment by bngarren** [comment]
Per f126571, the toggle button for Simple vs Nested content (**ToggleContentType** component) is now part of the layout of ContentInput. Also, updated some documentation. Issue is now closed.
